scripts/merge_tract_cat.py

```diff
@@ -93,6 +93,8 @@ def load_and_save_tract(repo, tract, filename, key_prefix='coadd', patches=None,
         except NoResults as e:
             print(e)
             continue
+        if len(patch_merged_cat) == 0:
+            continue
```

Comment:
Maybe add `print('empty patch')` so that it has a similar behavior like `print(e)` above?